ui/component/claimList/index.js

```diff
@@ -3,9 +3,13 @@ import ClaimList from './view';
 import * as SETTINGS from 'constants/settings';
 
 import { selectClientSetting } from 'redux/selectors/settings';
+import { selectOdyseeMembershipIsPremiumPlus } from 'redux/selectors/user';
+import { selectAdBlockerFound } from 'redux/selectors/app';
 
 const select = (state) => ({
   searchInLanguage: selectClientSetting(state, SETTINGS.SEARCH_IN_LANGUAGE),
+  hasPremiumPlus: selectOdyseeMembershipIsPremiumPlus(state),
+  adBlockerFound: selectAdBlockerFound(state),
```

Comment:
I think can remove now that the PremiumTile logic is at the caller side.
Clean up jsx side too.